exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillFactory.java

```diff
@@ -37,6 +38,24 @@
   protected final int major;
   protected final int minor;
 
+  static {
+    Properties prop = new Properties();
```

Comment:
Moved it to SecurityConfiguration.java. All unit tests are also passing and the application is also running fine.